site2/docs/client-libraries-node.md

```diff
@@ -27,7 +27,7 @@ Compatibility between each version of the Node.js client and the C++ client is a
 | 1.0.0          | 2.3.0 or later |
 | 1.1.0          | 2.4.0 or later |
 | 1.2.0          | 2.5.0 or later |
-| 1.3.0          | 2.7.0 or later |
+| 1.3.0          | 2.7.1 or later |
```

Comment:
I think 1.3.0 does not support this feature, we need to wait for the next release of the node client

Reply:
@tuteng OK, then I've removed `| 1.3.0          | 2.7.1 or later |` for both master and 2.7.2. PTAL.